### ISSUE TITLE
Add web support

### ISSUE THREAD

**Issue by MohamedRejeb** [M0Coding/Pokedex#3]
- Add implementation for web target

**Comment by satvikpendem** [comment]
Is web support able to be added now? I know it still requires a few extra steps but it seems like it's more stable now than in 2023.

**Comment by MohamedRejeb** [comment]
Yes I think it's time to add web support
I'm planning to move dependencies to `libs.versions.toml` as well and a few other changes.

**Comment by aritra-tech** [comment]
Hey @MohamedRejeb I was also thinking of adding Web support in my old project but how can we add the `wasmJsMain` module on our old project?

**Comment by MohamedRejeb** [comment]
Hi @aritra-tech 
If you don't have platform-specific code all you need to do is to add `wasmJs` target in gradle file. Also you need to check if all your dependencies are supporting `wasmJs`

**Comment by aritra-tech** [comment]
Okay got it.
Thanks @MohamedRejeb

**Comment by shriharsha-bhagwat** [comment]
Yes would be nice to build a web solution to this awesome repo! Are there any blockers such as database library etc?
